docs/Users_Guide/reformat_stat_data.rst

```diff
@@ -46,160 +53,634 @@ Plots that are generated from the command line require a YAML configuration file
+
+         - stat_bcl
+
+            - lower level bootstrap confidence limit
+
+         - stat_bcu
+
+            - upper level bootstrap confidence limit
+
+       - The original format is converted from wide format (where each row corresponds to a unique
+         set of common stat column values) to longer format (where there are multiple rows
+         corresponding to the same common stat column values).
+
+           - The common stat columns are common to all line types for point-stat, grid-stat, and ensemble-stat (model, vx mask, description, fcst lead, etc.)
+             as described in the MET User's Guide:
+
+              - table 11.1 (point-stat)
+              - table 12.1 (grid-stat)
+              - table 13.1 (ensemble-stat)
+
+
+
+         .. dropdown:: Unformatted Example:
+
+            .. literalinclude:: ./figure/grid_stat_FV3_core_lsm1_020000L_20190521_020000V.stat
+
+         - In the unformatted example:
+
+           - each row represents unique data (tidy data)
+           - the number of columns is different for each row, due to each row representing a different linetype
+             (e.g. DMAP, NBRTC)
+           - numerous columns are unlabelled
+
+
+         .. dropdown:: Reformatted Example (truncated):
+
+            ..  literalinclude:: ./figure/reformatted_dmap_for_lineplot_output.txt
 
-**Reformatting for
-line, bar, box, contour, performance diagram, revision box, revision series, and taylor diagram**
 
-After reformatting, the .stat data contains the following columns (replacing the unlabelled
-columns in the original .stat file and converting from wide to long format):
+         -  In the reformatted example:
 
-Two columns for statistics
+            - reformatted data file contains only one linetype (in this example DMAP), as requested in the configuration file
 
-- stat_name
+            - multiple rows have duplicate common stat column values, resulting in more rows
 
-- stat_value
+              - note that for the rows containing the first **fy** and **oy** stat_names, the common stat columns of
+                version,..., linetype, and total are the same.  The unique data has been replicated and separated
+                based on stat_name and stat_value.
 
-Four columns for confidence levels (normal and bootstrap)
+    .. dropdown::  by all common stat columns AND linetype-specific columns from MET
 
-- stat_bcl
+       - Used by the following plot types and linetypes in METplotpy:
 
-    - upper level bootstrap confidence limit
+         - **scatter** plot
 
-- stat_bcu
+           - **MPR** linetype
+           - **DMAP** linetype
 
-    - lower level bootstrap confidence limit
 
-- stat_ncl
+         - unlabelled columns in the original .stat file are placed under the linetype-specific column names
 
-    - lower level normal confidence limit
+           - refer to the MET User's Guide for point-stat, grid-stat, ensemble-stat, and tc-pairs  for the column names
```

Comment:
```suggestion
           - refer to the MET User's Guide for Point-Stat, Grid-Stat, Ensemble-Stat, and TC-Pairs  for the column names
```

Reply:
suggested changes made during an edit of the file (outside of this interface)